components/list/list-item-drag-image.js

```diff
@@ -0,0 +1,126 @@
+import '../colors/colors.js';
+import '../icons/icon.js';
+import '../inputs/input-checkbox.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { bodySmallStyles } from '../typography/styles.js';
+import { formatNumber } from '@brightspace-ui/intl/lib/number.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class ListItemDragImage extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * @ignore
+			 */
+			count: { type: Number }
+		};
+	}
+
+	static get styles() {
+		return [ bodySmallStyles, css`
+			:host {
+				display: block;
+				height: 70px;
+				left: -10000px;
+				position: absolute;
+				width: 340px;
+				z-index: 0;
+			}
+			:host([hidden]) {
+				display: none;
+			}
+			:host([dir="rtl"]) {
+				left: 0;
+				right: -10000px;
+			}
+			.first, .second, .third {
+				background-color: white;
+				border: 1px solid var(--d2l-color-mica);
+				border-radius: 4px;
+				box-sizing: border-box;
+				height: 100%;
+				position: absolute;
+				width: 100%;
+			}
+			.first {
+				align-items: start;
+				display: flex;
+				padding: 16px 8px;
+			}
+			.second {
+				margin-inline-start: 6px;
+				margin-top: 6px;
+				z-index: -1;
+			}
+			.third {
+				margin-inline-start: 12px;
+				margin-top: 12px;
+				z-index: -2;
+			}
+			.text {
+				width: 100%;
+			}
+			.line-1 {
+				background-color: var(--d2l-color-sylvite);
+				border-radius: 0.2rem;
+				height: 24px;
+				margin-bottom: 4px;
+				width: 100%;
+			}
+			.line-2 {
+				background-color: var(--d2l-color-sylvite);
+				border-radius: 0.2rem;
+				height: 16px;
+				width: 25%;
+			}
+			d2l-input-checkbox {
+				line-height: 0;
+				margin: 0;
+				margin-inline-start: 16px;
+			}
+			.count {
+				background-color: var(--d2l-color-celestine);
+				border-radius: 0.7rem;
+				box-sizing: border-box;
+				color: white;
+				left: 26px;
+				min-width: 1.4rem;
+				padding: 0.2rem 0.4rem;
+				position: absolute;
+				text-align: center;
+				top: 30px;
+				z-index: 1;
+			}
+			:host([dir="rtl"]) .count {
+				left: 14px;
+			}
+		`];
+	}
+
+	constructor() {
+		super();
+		this.count = 0;
+	}
+
+	render() {
+		return html`
+			<div class="first">
+				<div class="count d2l-body-small">${formatNumber(this.count)}</div>
```

Comment:
Interesting that we're not using the `<d2l-count-badge>` here since it does all this for us (but is grey or orange) -- I'm guess that was a design decision.

Reply:
I'll talk to Jeff & Noah about it.  We could introduce a variable to override the color if needed.

Reply:
Sounds good. Just seemed weird for us to have gone through and changed a bunch of places that had their own "coloured circle with a number in it" to use count badge and then re-introduce a new variant ourselves. :)

Reply:
Closing the loop on this - for now this will be a one-off since we don't want to introduce a color override or new `type` value on `d2l-count-badge`.  

Aside, when I first tried this it didn't show up because all the internals of the element being used needs to be updated before `e.dataTransfer.setDragImage` is called, and it must be done synchronously with the `dragstart` event or no drag image will be created at all. I think this timing issue could be resolved by always rendering `d2l-list-item-drag-image` for a draggable item, instead of waiting until `dragstart`.